File: sprint3/By languages/C#/lesson2code1.cs
using System;

class Program {
    static void stairs_builder(int n) {
        // build 1 step
        Console.WriteLine($"Осталось построить {n} ступеней.");
        stairs_builder(n - 1);
    }

    static void Main(string[] args) {
        stairs_builder(5);
    }
}

File: sprint3/By languages/go/lesson2code1.go
package main

import "fmt"

func stairsBuilder(n int) {
    // build 1 step
    fmt.Printf("Осталось построить %d ступеней.\n", n)
    stairsBuilder(n - 1)
}

func main() {
    stairsBuilder(5)
}
